test_scenarios/kubernetes/3idx1sh1cm-pvc/splunk-master-pvc.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
```

Comment:
This is super pedantic, but in the interest of consistency, I prefer to match labels across the board. So in the example PVCs, I'd use the same labels as the deployment that's using it. Especially in this case where the PVC is targeted to that specifc deployment.

Reply:
got an example?